source/blog/2019-02-14-octane-guides.md

```diff
@@ -0,0 +1,59 @@
+---
+title: 'Community Update: Octane Documentation'
+author: Melanie Sumner
+tags: Recent Posts, 2019, Octane
+responsive: true
+---
+
+The preview of the Octane edition of Ember is coming soon, and with all this work comes a lot of documentation. Since this requires coordination of many moving parts, we want to outline the approach we're taking, and outlining some ways that the community can contribute. 
+
+## Preparation
+
+In order to prepare for the new edition, the Learning Core Team spent time figuring out how to best transition the existing docs in such a way that was consistent with iterative innovation. 
+
+One of the goals of improving the guides in general was to improve the learning flow in the guides (see [RFC #431](https://github.com/emberjs/rfcs/pull/431)). As such, we determined that a change to the table of contents was in order.   
+
+## Updating the guides
+
+As per the usual standard, we have planned out this work to be able to complete it in an iterative fashion. This was particularly important because we are updating guides as new features for the Octane release are merged into Ember. 
+
+### Phases
+
+This is roughly the planned phases of the documentation work.
+
+Phase 1, Octane Guides, MVP: (see [Tracking issue](https://github.com/ember-learn/guides-source/issues/394))
+
+1. Create octane branch of the guides for this work
+1. Reorder the table of contents (as per the RFC)
+1. Add in placeholders for any new sub-sections
+1. Add or update content in each sub-section (prose, code samples) [https://github.com/ember-learn/guides-source/issues/394](https://github.com/ember-learn/guides-source/issues/394)
+1. Review, Edit & re-arrange sub-sections before preview launches
+1. Add re-direct URLs and remove old content
+
+Then the Octane guides will be considered in MVP status and we will move forward to the next phase, which will include all of the sections that were identified in the table of contents RFC but were not included in the MVP. 
+
+Through all of these phases, a preview of the Octane edition guides will be used to ensure that the documentation is staying on track with the intended features. 
+
+While new content is being written and older content is being updated, **no existing content will be completely deleted.** Only _after_ we have fully reviewed the content and added the re-directs will outdated content be removed. This means that in the interim, some content may be available if the user has the URL, but it won't be linked from the table of contents. 
+
+### Challenges
+As with any ambitious undertaking, there are some related challenges. Here are a few we are currently facing: 
+
+- converting guides from classic syntax to new syntax
```

Comment:
```suggestion
- Converting guides from classic syntax to new syntax;
```

Reply:
why are you putting semicolons at the end of each bullet point?